torch/csrc/distributed/rpc/message.h

```diff
@@ -62,7 +64,11 @@ class TORCH_API Message final {
   Message& operator=(Message&& rhs) &;
   void swap(Message& rhs) noexcept;
 
+  // Destructively retrieves the payload.
+  std::vector<char>&& movePayload();
```

Comment:
Consider making this && self qualified.

```
std::vector<char>&& movePayload() &&;
```

Reply:
@smessmer In many situations, return an rvalue reference is wrong. Do you know if it's wrong in this case? I'm brain-frozen right now...